services/ui-src/src/components/statusing/ChildRow.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+// components
+import { ChildRow } from "./ChildRow";
+import { TableRow } from "./TableRow";
+import { Table } from "@chakra-ui/react";
+// utils
+import { useStore } from "utils";
+import {
+  mockChildRowPage,
+  mockUseStore,
+  RouterWrappedComponent,
+} from "utils/testing/setupJest";
+
+jest.mock("utils/state/useStore");
+const mockedUseStore = useStore as jest.MockedFunction<typeof useStore>;
+
+const childRowComponent = (
+  <RouterWrappedComponent>
+    <Table>
+      <TableRow page={mockChildRowPage} rowDepth={2} />
+      {mockChildRowPage.children?.map((child) => (
+        <ChildRow key={child.path} page={child} rowDepth={2} />
+      ))}
+    </Table>
+  </RouterWrappedComponent>
+);
+
+describe("Test ChildRow", () => {
+  beforeEach(() => {
+    mockedUseStore.mockReturnValue(mockUseStore);
+    render(childRowComponent);
+  });
+  test("Check that childrow renders", () => {
+    expect(
+      screen.getByText("State or Territory-Specific Initiatives")
```

Comment:
If available, I like to use `getByRole` instead of `getByText` in case there's a chance the text could get rendered somewhere else in the component than where I'm expecting. For this you could use `getByRole` and `gridcell`.